fetch.bs

```diff
@@ -1938,6 +1957,10 @@ not always relevant and might require different behavior.
 "<code>client</code>" or an <a for=/>origin</a>. Unless stated otherwise it is
 "<code>client</code>".
 
+<p>A <a for=/>request</a> has an associated
+<dfn export for=request id=concept-request-navigation-initiator-origin>top-level navigation initiator origin</dfn>,
```

Comment:
Leave out the ID please. For new `<dfn>`s we generally let Bikeshed generate the ID.